src/main/resources/application.properties

```diff
@@ -9,5 +9,5 @@ spring.kafka.bootstrap-server=${S_KAFKA_BOOTSTRAP_HOST:${default.kafka.bootstrap
 
 # flyway properties - https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html#data-migration-properties
 
-#Cron job running every 12 hours
-job.cron.export.product.list=0 0 */12 * * *
\ No newline at end of file
+#Cron job running every 24 hours
+job.cron.export.product.list=0 0 */24 * * *
```

Comment:
I think cron expression should be like this 0 0 0 * * ?

Reply:
Changed to 0 0 2 * * *